.github/workflows/pr.yml

```diff
@@ -12,15 +12,21 @@ jobs:
   lint-test:
     name: Validate code and charts
     runs-on: ubuntu-18.04
+    container: 
```

Comment:
Could you please also remove envs `go_version` and `linter_version`? ^^

Reply:
@yurkov-anton Can you check? I also know another way how to separate the version(by `awk`, `cut`), but it might suit too..